### PR TITLE
Blade Sample Tests

### PR DESCRIPTION
**Pull request by lawrence-lee**
This assumes that the jars are already built. I dont know if we can build both gradle and maven packages from the root level. At least I dont know if we can in gradle.

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,34 @@
+apply plugin: 'java'
+
+FileTree bndtoolsGradleTree = fileTree(
+	dir: 'bndtools/', include: '**/dist/bundles/**/*.jar', exclude: '**/*latest.jar')
+FileTree gradleTree = fileTree(dir: 'gradle/', include: '**/libs/*.jar')
+FileTree liferayGradleTree = fileTree(dir: 'liferay-gradle/', include: '**/libs/*.jar')
+FileTree mavenTree = fileTree(dir: 'maven/', include: '**/target/*.jar')
+
+task bndtoolSizeTest <<{
+	assert 25 == bndtoolsTree.size()
```
Comment (gamerson) [inline build.gradle]: bndtoolsTree variable doesn't exist, its called "bndtoolsGradleTree" I think.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,34 @@
+apply plugin: 'java'
```
Comment (gamerson) [inline build.gradle]: no need to apply java plugin

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+FileTree bndtoolsGradleTree = fileTree(
```
Comment (gamerson) [inline build.gradle]: Lets call these fileTrees like this naming scheme

FileTree bndtoolsOutputFiles
FileTree gradleOutputFiles
etc

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	dir: 'bndtools/', include: '**/dist/bundles/**/*.jar', exclude: '**/*latest.jar')
```
Comment (gamerson) [inline build.gradle]: No need to add this to a 2nd line for source formatting, you can keep it all on the same line.

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+	assert [] == collection1
+
+	def collection2 = liferayGradleTree.files.name.sort() - bndtoolsGradleTree.files.name.sort()
+	assert [] == collection2
+
+	def collection3 = liferayGradleTree.files.name.sort() - mavenTree.files.name.sort()
+	assert [] == collection3
+}
+
+task gradleSizeTest <<{
```
Comment (gamerson) [inline build.gradle]: Lets put all of the asserts for the file list size into the same task called "outputFilesSizeTest" or something like that.

**Comment by gamerson** [comment]
Please resend, thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by lawrence-lee** [comment]
@gamerson Thanks for the feedback. Will do!

**Pull request closed without merging**